.standard_todo.yml

```diff
@@ -0,0 +1,6 @@
+# Auto generated files with errors to ignore.
```

Comment:
An alternative to adding a TODO file is locking `standard` to a specific version. I added this because it's outside the scope of this PR.